.ci/sonar.sh

```diff
@@ -0,0 +1,27 @@
+#set checkstyle sonar profile
```

Comment:
please make this file as shell file (not a source) to allow multiple execution on running Sonar instance.
please make file executable.